File: Python/Exercicio10.py
"""
Tendo como dados de entrada a altura e o sexo de uma pessoa, construa
um algoritimo que calcule seu peso ideal, utilizando as seguintes formulas:
    Para homens: (72.7 * altura) - 58
    Para Mulheres: (62.1 * altura) - 44.7
"""
sexo = input("Informe seu sexo 'F' para feminino ou 'M' para masculino: ")
altura = float(input("Informe sua altura: "))

if sexo == 'M' or sexo == 'm':
    peso_ideal = (72.7 * altura) - 58
elif sexo == 'F' or sexo == 'f':
    peso_ideal = (62.1 * altura) - 44.7
else:
    print("Opção Invalida!")

print("O seu peso ideal é: {:.2f}".format(peso_ideal))

File: Java/Exercicio10.java
/*
Tendo como dados de entrada a altura e o sexo de uma pessoa, construa
um algoritimo que calcule seu peso ideal, utilizando as seguintes formulas:
    Para homens: (72.7 * altura) - 58
    Para Mulheres: (62.1 * altura) - 44.7
*/

package Java;

import java.util.Scanner;

public class Exercicio10 {

	public static void main(String[] args) {
		
		System.out.println("Informe sua altura: ");
		float altura = new Scanner(System.in).nextFloat();
		
		System.out.println("Informe o sexo F para feminino e M para masculino:");
		String sexo = new Scanner(System.in).nextLine();
		
		if(sexo == "M" || sexo == "m") {
			double pesoIdeal = (72.7 * altura) - 58;
			System.out.println("Seu peso ideal e: " + String.format("%.2f", pesoIdeal));
		}
		else if(sexo == "F" || sexo == "f") {
			double pesoIdeal = (62.1 * altura) - 44.7;
			System.out.println("Seu peso ideal e: " + String.format("%.2f", pesoIdeal));
		}
		
	}
}